src/pages/api/nextjs/revalidate-project.page.ts

```diff
@@ -41,6 +42,10 @@ export default async function handler(
   console.info(`[Next.js] Revalidating ${path}`)
   await res.revalidate(path)
 
+  // Update sepana database whenever a project needs revalidating
+  // Note: This won't work if it takes too long for project data to become available in the subgraph
+  await axios.get('/api/sepana/update')
```

Comment:
In that case, is it worth `try-catching` this? or is it preferred to let it error?

Reply:
it won't _error_, just won't actually write any new data. comment could be more specific

Reply:
From what I read recently, serverless nextjs api should be quick. How long does update usually take? I think we have a cap of 60s according to:
https://vercel.com/docs/concepts/limits/overview#general-limits

Reply:
Great point. maybe worth not even `await`ing this at all? just fire it off?

Reply:
sry thought i responded to this. no endpoints should take more than a couple seconds (even populating the db from empty takes less than 10s, but that'll likely won't ever happen in production)